N25: Liminal Lab 000 (#762)
rust-gamedev/rust-gamedev.github.io: content/news/025/index.md

[bevy_easings]: https://github.com/mockersf/bevy_extra/tree/master/bevy_easings
[@rparrett]: https://github.com/rparrett

### Liminal Lab 000

[![Screenshot of Liminal Lab 000 showing a white-walled laboratory test chamber
with buttons on the floor, lights on the wall, and a dark cube levitating
overhead.](liminal_lab_000.png)][ll000-live]
_Click the image to play the game in your desktop browser!_

Liminal Lab 000 ([live version][ll000-live]) by [@pebaz] is a tiny,
minimalistic puzzle game with 1 puzzle designed around the concept of
[Liminal Spaces][liminality]. Liminal Spaces are usually abandoned,
transitional places where life once thrived. These spaces are somehow familiar
to the viewer but the viewer has never been there. The unsettling feeling of
being alone comes from the realization that the viewer does not belong in that
space but is merely passing through it.

Liminal Lab 000 was built using [Macroquad][macroquad], utilizes voxel

> rendering with 8x8x8 chunks, and is hosted on Google Firebase Hosting. More
> information about the game can be found on the initial
> [Reddit discussion][liminal-reddit].

Reviewer: ```suggestion
rendering with 8x8x8 chunks, and is hosted on Google Firebase Hosting.

_Discussions: [/r/rust_gamedev][liminal-reddit]_
```
